### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/project-generation.md`

### PR DESCRIPTION
**Pull request by Etuloser**
There is already exists https://github.com/tiangolo/fastapi/pull/3831 with the translation of this page.
But it seems dead and I decided to try to complete it.

### CODE REVIEW

File: docs/zh/docs/project-generation.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+## 全栈 FastAPI PostgreSQL
+
+GitHub: <a href="https://github.com/tiangolo/full-stack-fastapi-postgresql" class="external-link" target="_blank">https://github.com/tiangolo/full-stack-fastapi-postgresql</a>
+
+### 全栈 FastAPI PostgreSQL - 特性
+
+* 完整的 **Docker** 集成 (基于Docker)。
+* Docker Swarm 模式部署。
+* **Docker Compose** 集成和优化本地开发。
+* 使用 Uvicorn 和 Gunicorn 的 **生产就绪** Python Web 服务器。
```
Comment (wdh99) [inline docs/zh/docs/project-generation.md:16]: 生产就绪(Production-ready)我觉得改为'准生产'好些

File: docs/zh/docs/project-generation.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## 全栈 FastAPI PostgreSQL
+
+GitHub: <a href="https://github.com/tiangolo/full-stack-fastapi-postgresql" class="external-link" target="_blank">https://github.com/tiangolo/full-stack-fastapi-postgresql</a>
+
+### 全栈 FastAPI PostgreSQL - 特性
+
+* 完整的 **Docker** 集成 (基于Docker)。
+* Docker Swarm 模式部署。
+* **Docker Compose** 集成和优化本地开发。
```
Comment (wdh99) [inline docs/zh/docs/project-generation.md:15]: 针对本地开发的**Docker Compose** 集成和优化

File: docs/zh/docs/project-generation.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    * **基于标准**: 基于(并且完全兼容)关于API的开放标准: <a href="https://github。com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> 和 <a href="https://json-schema。org/" class="external-link" target="_blank">JSON Schema</a>。
+    * <a href="https://fastapi。tiangolo。com/features/" class="external-link" target="_blank">**许多其他特性**</a>， 包括自动表单验证、序列化、交互式文档、OAuth2 JWT令牌认证等。
+* 默认**安全密码**哈希。
+* **JWT token** 认证。
+* **SQLAlchemy** 模型 (独立于 Flask 扩展， 因此它们可以直接用于 Celery workers)。
+* 基础的用户模型 (根据需要进行更改和删除)。
+* **Alembic** 迁移。
+* **CORS** (跨域资源共享)。
+* 可以选择性地从后端导入和使用模型和代码的 **Celery** worker。
+* 基于 **Pytest** 的REST 后端测试，与Docker集成，因此您可以独立于数据库测试全部 API 交互。由于它在Docker中运行，所以每次它可以从头开始构建一个新的数据存储(因此您可以使用 ElasticSearch，MongoDB，CouchDB 或任何您想要的，只是测试 API 是否正常工作)。
```
Comment (wdh99) [inline docs/zh/docs/project-generation.md:32]: 只是测试 API 是否正常工作 -> 只需测试 API 是否正常工作

File: docs/zh/docs/project-generation.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+GitHub: <a href="https://github.com/tiangolo/full-stack-fastapi-couchbase" class="external-link" target="_blank">https://github.com/tiangolo/full-stack-fastapi-couchbase</a>
+
+⚠️ **警告** ⚠️
+
+如果您从头开始启动一个新项目，请查看这里的替代方案。
+
+例如，项目生成器<a href="https://github。com/tiangolo/full-stack-fastapi-postgresql" class="external-link" target="_blank">Full Stack FastAPI PostgreSQL</a> 可能是更好的选择，因为它正在活跃地维护和使用。并且包含了所有新的功能和改进。
+
+如果您仍然想使用基于Couchbase的生成器，它可能仍然可以很好地工作，并且如果您已经有一个生成的项目适合您的需求也是可以的。
```
Comment (wdh99) [inline docs/zh/docs/project-generation.md:65]: 我觉得改成这样容易理解：
如果您仍然想使用基于Couchbase的生成器，它可能仍然可以很好地工作，如果您已经用它生成了一个项目，也可以（你可能已经更新至满足需求）。

File: docs/zh/docs/project-generation.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+## spaCy 和 FastAPI 的机器学习模型
+
+GitHub: <a href="https://github.com/microsoft/cookiecutter-spacy-fastapi" class="external-link" target="_blank">https://github.com/microsoft/cookiecutter-spacy-fastapi</a>
+
+### spaCy和FastAPI的机器学习模型 - 特性
+
+* **spaCy** NER模型集成。
+* 内置**Azure Cognitive Search**请求格式。
+* 使用 Uvicorn 和 Gunicorn 的 **生产就绪** Python Web 服务器。
```
Comment (wdh99) [inline docs/zh/docs/project-generation.md:81]: 还是和前面一样建议将‘生产就绪’改为‘准生产’

File: docs/zh/docs/project-generation.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+* **Azure DevOps** Kubernetes (AKS) CI/CD 部署内置。
```
Comment (wdh99) [inline docs/zh/docs/project-generation.md:82]: 与80行保持一致，‘内置’放前面。内置**Azure DevOps** Kubernetes (AKS) CI/CD 部署。

**Comment by tiangolo** [comment]
📝 Docs preview for commit 070f7bb16f6d98b3b5cd0149c2c9eebfbf6953db at: https://649a2e2fb14d051ea004655a--fastapi.netlify.app

File: docs/zh/docs/project-generation.md (hunk 4 above)
Comment (Slijeff) [reply docs/zh/docs/project-generation.md:65]: ”它可能仍然可以很好地工作“ 感觉有点奇怪，是否可以改成 “应该不会有什么问题”

**Comment by xzmeng** [comment]
@Etuloser Hi, I noticed that this PR has been inactive for more than three months since the last review. Are you stilling paying attention to this PR? I don't mean to rush you, if you are ready to continue completing this PR, I can help review, if you are not free, I can create a new PR to continue this translation.

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Comment by alejsdev** [comment]
> @Etuloser Hi, I noticed that this PR has been inactive for more than three months since the last review. Are you stilling paying attention to this PR? I don't mean to rush you, if you are ready to continue completing this PR, I can help review, if you are not free, I can create a new PR to continue this translation.

Hi! this one was closed since was quite for a while and had no approval reviews, the old one PR (https://github.com/tiangolo/fastapi/pull/3831) was accepted because it had an approval review, if you want you can continue updating it. Thanks for your help! 🤓🚀

**Pull request closed without merging**